### PR TITLE
LINK-2038, LINK-2040 | allow multiple merchants and accounts

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Allows an organization to multiple accounts and merchants. Adds also "name" field for the organization account.

## Closes
[LINK-2038](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2038)
[LINK-2040](https://helsinkisolutionoffice.atlassian.net/browse/LINK-2040)

[LINK-2038]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2038?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[LINK-2040]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-2040?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr396-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr396-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr396-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## e2e tests result is __success__ for https://linkedcomponents-pr396-ui.dev.hel.ninja :laughing::tada::tada::tada:

**Review by mikkojamG — APPROVED** [review]

File: src/domain/organization/organizationForm/merchants/merchant/Merchant.tsx [hunk 1]
```diff
@@ -86,136 +86,113 @@ const Merchant: React.FC<Props> = ({
...
-          title={warning}
         />
       </FormRow>
       <FormRow className={inputRowBorderStyleIfHasInstance}>
         <Field
-          className={alignedInputStyle}
-          component={TextInputField}
-          disabled={!editable || !isEditingAllowed}
+          {...commonInputFieldProps}
           label={t(`organization.form.merchant.labelUrl`)}
```
Comment (harriris-vincit) [inline src/domain/organization/organizationForm/merchants/merchant/Merchant.tsx:160]: Merchants do not have a user-editable `url` field anymore in the back-end. The field can be removed from the front-end.

**Comment by terovirtanen** [comment]
**LINKEDCOMPONENTS-UI branch is deployed to platta: https://linkedcomponents-pr396-ui.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## e2e tests result is __failed__ for https://linkedcomponents-pr396-ui.dev.hel.ninja :crying_cat_face::anger::boom::boom:

File: src/domain/organization/organizationForm/merchants/merchant/Merchant.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/organization/organizationForm/merchants/merchant/Merchant.tsx]: Removed url from the merchant fields

**Pull request merged**